packages/ui/src/components/SelectInputContainer/index.tsx

```diff
@@ -268,6 +261,15 @@ function SelectInputContainer<
 
     const dropdownShownActual = dropdownShown && !dropdownHidden;
 
+    const availableOptions = options.filter(
+        (option) => !selectedOptions?.some(
+            (selectedOption) => optionKeySelector(
+                selectedOption,
+                0,
+            ) === optionKeySelector(option, 0),
+        ),
+    );
```

Comment:
Let's use a dict to filter these options. Also, add note what available options are.
Also, is this change correct for single select inputs?